apps/eaas-api/src/database/migrations/20231107162710_embalmer_has_client.ts

```diff
@@ -0,0 +1,13 @@
+import { Knex } from "knex";
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema.createTable("embalmer_has_client", (table) => {
```

Comment:
Thinking a better name might be `embalmer_clients` to indicate a one to many relationship. Technically this may support many to many (so `embalmers_clients` could work), but thinking in practice it will be one to many (clients won't use many embalmers).

Reply:
I prefer the "a_has_b" nomenclature since it makes the direction of "ownership" clearer. Maybe `embalmer_has_clients` to explicitly indicate the one-to-many relationship?

Reply:
I like `embalmer_has_clients` -- that makes sense!